.github/workflows/build.yaml

```diff
@@ -32,6 +32,8 @@ jobs:
       TAG: ${{ matrix.distro }}
 
     steps:
+      - uses: docker/setup-qemu-action@v1
+      - uses: docker/setup-buildx-action@v1
```

Comment:
Use our internal tools action

https://github.com/containerbase/buildpack/blob/0d1a61e374bbda2b552d708de1766be82700c56e/.github/workflows/build.yaml#L84-L87

Reply:
just to make sure I'm not missing anything:

It looks like internal-tools sets up the required environment to create arm64 builds, but I don't see a way to use it.
in the docker build function, there is no way that I can see to pass --platform=foo into buildx, even though the buildx environment looks like it can support cross building arm64.

Am I missing something?